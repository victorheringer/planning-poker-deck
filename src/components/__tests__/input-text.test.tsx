import { render } from "@testing-library/react";
import InputText from "../input-text";

describe("InputText", () => {
  it("should render correctly", () => {
    const tree = render(<InputText />).asFragment();
    expect(tree).toMatchSnapshot();
  });
});
