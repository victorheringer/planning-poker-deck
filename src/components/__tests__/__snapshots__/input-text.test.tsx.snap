// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputText should render correctly 1`] = `
<DocumentFragment>
  <input
    class="sc-bdvvtL grFbFg"
  />
</DocumentFragment>
`;
